.gitignore

```diff
@@ -0,0 +1 @@
+node_modules
```

Comment:
didn't work, which really was strange, yea. that's why i added it. just delete it after merge?